models/keep3r_network/keep3r_network_addLiquidityToJob.sql

```diff
@@ -0,0 +1,68 @@
+{{ config(
+      tags=['dunesql']
+      , alias = alias('etv_addliquiditytoJob')
```

Comment:
we universally use `lower_case_with_underscore` for naming standard, for files/directories/aliases/schemas/columns, can you ensure all here is updated to reflect that?